### ISSUE TITLE
Support doccano v1.x on Heroku

### ISSUE THREAD

**Issue by icoxfog417** [doccano/doccano#811]
Support doccano v1.x deployment by Heroku button.

**Comment by Virmli** [comment]
Any update regarding this question? Maybe some guid lines how to build v1.x manually on Heroku?

**Comment by leemengtaiwan** [comment]
naive questions: 
- where can we know the current version that is being deployed to Heroku?

**Comment by pjchungmd** [comment]
I'd like to jump on @leemengtaiwan 's question. When I clone and run Doccano from my machine, the version seems to be far less mature compared to when I run it using the Heroku deploy button. Why is that?

**Comment by Hironsan** [comment]
fixed by https://github.com/doccano/doccano/pull/1088

**Comment by louisguitton** [comment]
@leemengtaiwan @pjchungmd to your questions, Heroku is setup to use the Dockerfile from the main folder (https://github.com/doccano/doccano/blob/master/Dockerfile) as you can see from the config here https://github.com/doccano/doccano/blob/master/heroku.yml#L3 . 

On the other hand, the docker-compose.yml uses other Docker images (the ones in the `./app` and `./frontend` folders)

So basically you're just not running the same thing in Heroku and docker-compose.
The frontend that looks "older" is the django frontend based on templated HTML. The frontend that looks more "modern" is the Vue.js frontend.
 
It can maybe be confusing when you're not confident with docker, but what @Hironsan has pulled off is to basically revamp the full UI independently from the django app, which is a big achievement.

**Comment by louisguitton** [comment]
> fixed by #1088

Aaand just like that, @Hironsan drops a big one
I've tested it and am a happy user :) 

```sh
docker build . -t doccano/vue
docker run -it -p 8000:8000 doccano/vue
open http://0.0.0.0:8000
```

![image](https://user-images.githubusercontent.com/7823843/100868955-9f794700-349c-11eb-844d-c0e7cc68b78f.png)
